README.md

```diff
@@ -436,6 +476,15 @@ Be sure that VTK has been built with *OpenImageDenoise* support (`VTKOSPRAY_ENAB
   * If no formats have working thumbnails, then it is an issue with the f3d.thumbnailer file
 
 ## Windows
+> After installing F3D or registering the shell extension, my explorer is broken
+
+Unregister the shell extension by running:
+
+```
+cd C:\path\to\f3d\bin\
+regsvr32 /u F3DShellExtension.dll
```

Comment:
did you try this to fix the issue ? @Meakk

Reply:
Nop I had to use ShellExView to disable it